source/examples/xgboost-azure-mnmg-daskcloudprovider/notebook.ipynb

```diff
@@ -0,0 +1,1422 @@
+{
```

Comment:
<p>This seems like an unecessarily complex way to scale workers. Especially using <code>has_what</code> as a proxy for the number of workers. I think just calling <code>cluster.scale</code> and <code>client.wait_for_workers</code> is enough. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/deployment/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>